content/platforms/release-notes/k8s-6-2-4-tbd-2021-09.md

```diff
@@ -0,0 +1,135 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.20-<tbd> (Sept 2021)
+linktitle: 6.2.4-<tbd> (Sept 2021)
+description: Release notes for version 6.2.4-<tbd>
+weight: 67
+alwaysopen: false
+categories: ["Platforms"]
+---
+## Overview
+
+The Redis Enterprise [K8s 6.2.4-55](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.2.4-55) release provides support for the [Redis Enterprise Software release 6.2.4]({{<relref "/rs/release-notes/rs-6-2-4-august-2021.md">}}) and includes several enhancements and bug fixes.
```

Comment:
The K8s 6.2.4-55 link currently gives me a 404, but maybe this is just a GitHub permissions issue on my part though?